packages/aws-client/index.js

```diff
@@ -1,14 +1,12 @@
 exports.client = require('./client');
-exports.cloudformation = require('./cloudformation');
-exports.dynamo = require('./dynamo');
+exports.cloudformation = require('./CloudFormation');
```

Comment:
Seems like the exports in this file should match the case of the things that they're exporting. `s3` -> `S3`, for example.